contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,99 @@
+* Number of purchases, since more people arriving at the checkout page should mean more purchases
+
+This also makes it easier to investigate the experiment results, especially when they are not what we expected. For example:
+* If the color change didn't increase button clicks, there may be a different issue on the page.
+* If the number of button clicks increased but the number of purchases did not, there may be an issue with the checkout page.
+
+## 2. Not monitoring counter metrics
+
+Counter metrics measure unintended negative side-effects in your experiments. If you do not monitor them, you may accidentally roll out changes that result in a worse user experience. 
+
+For example, say you're testing a change in the number of marketing emails you send to your users. While the change may increase the number of visitors to your site, it may also increase the number of users who unsubscribe from your mailing list. In this case, the unsubscribe rate is a good counter metric to keep track of.
+
+## 3. Not testing experiment setup with a small rollout
+
+Sometimes we are so eager to get results from our experiments that we jump straight into testing it with all our users. This may be okay if everything is set up correctly, but if you've made a mistake, you may not be able to rerun your experiment. Let's look at why this is:
+
+Imagine you're an experiment with a 50/50 split between control and test. You roll out the experiment to all your users, but a day after launch you notice that your change is causing the app to crash for all test users. You immediately stop the experiment and fix the root cause of the crash. However, restarting the experiment now will produce unreliable results since many users have already seen your change.
+
+To avoid this problem, you should first test your experiment with a small rollout (e.g., 5% of users) for a few days. Once you're confident everything works correctly, you can start the experiment with the remaining users.
+
+## 4. Including unaffected users in your experiment
+
+Another common mistake in A/B testing is counting users in your experiment who aren't actually affected by the change you're testing. This can happen if you're not careful in how and when you assign users to the control and test groups. It dilutes your experiment results, making it harder to determine the impact of your changes. 
+
+To see how, let's look at an example: Say you're testing a new feature in your app that rewards users for completing a certain action. You mistakenly included users who have already completed the action in the experiment. Since they are not affected by the change, any metrics related to this action do not change, and thus the results for this experiment may not show a statistically significant change.
+
+To avoid this mistake, make sure to first filter out ineligible users in your code before including them in your experiment. Below is an example of how to do this:
+
+```javascript
```

Comment:
We should use PostHog in the code example, even if it is just `posthog.isFeatureEnabled('keyword')`